guide/payments/transactions.md

```diff
@@ -32,18 +32,67 @@ transactions, it's important for designers to understand these differences when
 Illustration sources
 
 - https://www.figma.com/file/Da2FpXLqY2BX7WN4mQFNZo/Inputs-and-Outputs-Christoph-Copy?node-id=104%3A138
+- https://www.figma.com/file/GPyzunxlHOPXILudhrkZRi/Bitcoin-Transactions-and-Payments-Charts?node-id=0%3A1
 
 -->
 
 # Transactions overview
 
-This page is meant to help you with understanding how bitcoin transactions get created and processed by the Bitcoin network. Bitcoin is a peer-to-peer push-payment system. This means that you can send, or push, bitcoin to any address at your will, at any time, without passing through a trusted third party.
+This page is meant to help you with understanding how bitcoin transactions get created and processed by the Bitcoin and Lightning networks. Bitcoin is a peer-to-peer push-payment system. This means that you can send, or push, bitcoin at any time, without passing through a trusted third party.
 
-This is radically different from the traditional financial system, where it is often possible for others to *pull* and withdraw money from your account (utility companies, financial institutions, merchants, etc.). When you make a payment, it will pass through systems that might delay, control or block the payment.
+This is radically different from the traditional financial system, where it is often possible for others to pull and withdraw money from your account (utility companies, financial institutions, merchants, etc.). When you make a payment, it will pass through systems that might delay, control or block the payment.
 
-## Transaction lifecycle
+## Comparing Bitcoin and Lightning
 
-Let's lay out the entire payment process.
+| Bitcoin          | Lightning |
+|-------------|------------------|
+| Users send *[transactions]({{'/guide/glossary/#transaction' | relative_url}})*. | Users send *[payments]({{'/guide/glossary/#payment' | relative_url}})*.  |
```

Comment:
I find it super unintuitive to distinguish transactions and payments by which layer they are on. To me, those words mean 95% the same thing. A distinction I could see being made is that a transaction is the technical thing that happens, while a payment is the thing users make.

Reply:
Especially weird since the section is called `Payments`. The tech primer also uses the term `transactions` when referring to Lightning.

Reply:
Yeah I agree, could just remove the first line and change this section name to `Comparing Bitcoin and Lightning payments`.

Could distinguish how Bitcoin is more a settlement layer where as Lightning is more for P2P payments.

Reply:
We had a [Slack discussion about this point](https://bitcoindesign.slack.com/archives/C015856BDME/p1639994002171500).

Let me summarize briefly what I was trying to convey in that thread. Consider this: "Alice pays Bob". We could provide two different descriptions of what is happening in that sentence. One is the **user experience description**. The other is the **technical description**. For the UX description, I feel comfortable using the terms payment and transaction interchangeably. However, for the tech description, the 2 terms should not be used interchangeably. They are very different concepts.

[In this Figma file](https://www.figma.com/file/i5BpraIpOqpqKDRthoxilX/Transactions-and-Payments-Examples?node-id=0%3A1), I have tried to lay out the differences visually.

Since this page is providing more of a technical overview, I think it might be a good idea to use tech descriptions and not UX descriptions.

I tried to update and make this idea a little more clear in 513973c.

Thoughts?